### ISSUE TITLE
Would it be possible to create a readonly version, as to reduce the scopes needed?

### ISSUE THREAD

**Issue by DavyLandman** [GAM-team/got-your-back#464]
My admin is being a bit iffy about granting write rights to the organization, just for an automated backup.

I was looking at the scopes and I noticed that `https://www.googleapis.com/auth/gmail.readonly` exists, would it be possible to make a version that only does read-only requests? or is that a large undertaking?

**Comment by hudcap** [comment]
When you set it up for the first time, I'm pretty sure it asks for exactly which scopes you want to authorize. Can't promise, because I set this up ages ago...

**Comment by hudcap** [comment]
Yup: https://github.com/GAM-team/got-your-back/blob/493ddf9d1cc0229577f043c36c267a87e1e84f37/gyb.py#L470-L489

**Comment by DavyLandman** [comment]
aah, my bad, I should have used the force and read the source.

**Comment by DavyLandman** [comment]
@hudcap 

I now remember, I was reading the wiki and here is says:

> For API Scopes, enter exactly:
> `https://mail.google.com/,https://www.googleapis.com/auth/apps.groups.migration,https://www.googleapis.com/auth/drive.appdata`

So that what got me on the wrong foot. I'll see if it indeed also works for service accounts to be on `.readonly` scope.
